src/app/components/bitcoin-fees-list/components/fees-list-item.tsx

```diff
@@ -19,22 +19,24 @@ export function FeesListItem({
   isSelected,
   ...props
 }: FeesListItemProps) {
+  // #4476 TODO visually inspect this
   return (
     <styled.button
       _hover={{ background: 'accent.component-background-hover' }}
       border={isSelected ? '4px solid' : '1px solid'}
       borderColor="accent.action-primary-default"
       borderRadius="lg"
+      // #4476 TODO add tokens for boxShadow
       boxShadow="0px 1px 2px rgba(0, 0, 0, 0.04)"
       data-testid={SharedComponentsSelectors.FeesListItem}
-      px="base"
-      py="extra-loose"
+      px="space.04"
+      py="space.06"
```

Comment:
Thx, I think I missed these. 👍

Reply:
No worries. There's so many. I wonder if it would be more efficient to leave them alone, then do a big `find+replace` at the end when we have moved all the components. 

I guess no harm to change ones as we go too